pkg/controller/shared/authorization_informers.go

```diff
@@ -35,7 +35,7 @@ func (f *clusterPolicyInformer) Informer() cache.SharedIndexInformer {
 		return informer
 	}
 
-	lw := f.customListerWatchers.GetListerWatcher(kapi.Resource("clusterpolicies"))
+	lw := f.customListerWatchers.GetListerWatcher(authorizationapi.Resource("clusterpolicies"))
```

Comment:
I was struggling here, which one we should actually use, that's why I'm raising this now.